lib/pf/cmd/pf.pm

```diff
@@ -0,0 +1,97 @@
+package pf::cmd::pf;
+=head1 NAME
+
+pf::cmd::pf add documentation
+
+=head1 SYNOPSIS
+
+pfcmd <command> [options]
+
+ Commands
+  checkup                     | perform a sanity checkup and report any problems or warnings
+  class                       | view violation classes
+  config                      | query, set, or get help on pf.conf configuration paramaters
+  configfiles                 | push or pull configfiles into/from database
+  floatingnetworkdeviceconfig | query/modify floating network device configuration parameters
+  fingerprint                 | view DHCP Fingerprints
+  fixpermissions              | fix permissions of files
+  graph                       | trending graphs
+  history                     | IP/MAC history
+  import                      | bulk import of information into the database
+  ifoctetshistorymac          | accounting history
+  ifoctetshistoryswitch       | accounting history
+  ifoctetshistoryuser         | accounting history
+  interfaceconfig             | query/modify interface configuration parameters
+  ipmachistory                | IP/MAC history
+  locationhistorymac          | Switch/Port history
+  locationhistoryswitch       | Switch/Port history
+  lookup                      | node or pid lookup against local data store
+  manage                      | manage node entries
+  networkconfig               | query/modify network configuration parameters
+  node                        | node manipulation
+  nodeaccounting              | RADIUS accounting information
+  nodecategory                | nodecategory manipulation
+  nodeuseragent               | View User-Agent information associated to a node
+  person                      | person manipulation
+  reload                      | rebuild fingerprint or violations tables without restart
+  report                      | current usage reports
+  schedule                    | Nessus scan scheduling
+  service                     | start/stop/restart and get PF daemon status
+  switchconfig                | query/modify switches.conf configuration parameters
+  switchlocation              | view switchport description and location
+  traplog                     | update traplog RRD files and graphs or obtain switch IPs
+  trigger                     | view and throw triggers
+  ui                          | used by web UI to create menu hierarchies and dashboard
+  update                      | download canonical fingerprint or OUI data
+  useragent                   | view User-Agent fingerprint information
+  version                     | output version information
+  violation                   | violation manipulation
+  violationconfig             | query/modify violations.conf configuration parameters
```

Comment:
I would rather we add these as they are ready.
Otherwise there will inevitably be one that is not implemented.